app/controllers/shared/api.php

```diff
@@ -87,6 +87,25 @@
     }
 };
 
+App::setResource('prepareUserSearch', function () {
```

Comment:
We do not use resource for global functions, this could become a very messy pattern over time. For this scenario, using database filters would work great.